docs/notable-contributors.md

```diff
@@ -0,0 +1,17 @@
+# All Notable Contributors
+
+This page contains all Notable Contributors for the Linux on T2 project. \
+If there is someone missing that you think should be on this list, send a PR on the [wiki repository.](https://github.com/t2linux/wiki)
+
+- aunali1 (Arch Linux and Kernel Module work) [https://github.com/aunali1](https://github.com/aunali1)
+- JPyke3 (Manjaro) [https://github.com/JPyke3](https://github.com/JPyke3)
+- marcosfad (Ubuntu) [https://github.com/marcosfad](https://github.com/marcosfad)
+- MCMrARM (MacBook Bridge / T2 Linux Driver) [https://github.com/MCMrARM](https://github.com/MCMrARM)
+- mikeeq (Fedora) [https://github.com/mikeeq](https://github.com/mikeeq)
+- networkException (Wiki maintenance and `mbpfan` fork for T2) [https://github.com/networkException](https://github.com/networkException)
+- Redecorating (Wiki maintenance and providing compiled kernels with Corellium patch for wifi) [https://github.com/Redecorating](https://github.com/Redecorating)
```

Comment:
I guess the correlium part can be removed?

Reply:
Fixed in commit 2a97094